src/components/TreebankService/TreebankDestinationConfig.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
You don't have to worry about block comments for functions. The rest of the code base doesn't use them and I think it makes the most sense to keep it all consistent. But if it helps you to write them while developing the code I don't want to stop you!

Reply:
happy to remove them. We use them in Alpheios so it was just habit.